package.json

```diff
@@ -14,30 +14,31 @@
   "license": "MIT",
   "main": "index.js",
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=6.9.0"
+    "node": ">=16.0.0",
+    "npm": ">=8.0.0"
   },
   "dependencies": {
-    "@babel/core": "~7.17.12",
-    "@babel/preset-env": "~7.17.12",
-    "@spryker/oryx": "^1.4.0",
-    "autoprefixer": "~8.3.0",
-    "babel-loader": "~8.2.5",
-    "core-js": "~2.5.5",
+    "@babel/core": "~7.21.3",
+    "@babel/preset-env": "~7.20.2",
+    "autoprefixer": "~9.8.8",
+    "babel-loader": "~8.3.0",
+    "colors": "~1.4.0",
+    "core-js": "~2.6.12",
     "css-loader": "~2.1.1",
-    "es6-promise": "~4.0.5",
+    "css-minimizer-webpack-plugin": "~4.2.2",
+    "fast-glob": "~3.2.12",
     "file-loader": "~1.1.11",
-    "fs-extra": "~10.0.0",
+    "fs-extra": "~10.1.0",
     "imports-loader": "~0.8.0",
-    "mini-css-extract-plugin": "~0.4.0",
-    "optimize-css-assets-webpack-plugin": "~5.0.8",
-    "postcss-loader": "~2.1.4",
+    "mini-css-extract-plugin": "~2.7.2",
+    "moment": "~2.29.4",
```

Comment:
Suggest to replace huge momentjs dependency with something smaller - [dayjs](https://github.com/iamkun/dayjs) for example.